### PR TITLE
Updating Anaconda installation instructions for mac

### PR DESCRIPTION
**Pull request by pjh5**

### CODE REVIEW

File: _docs/getting-started/mac.md [hunk 1]
```diff
@@ -113,7 +111,94 @@ Change to a different folder and test Caffe2 again. If you are using Anaconda or
...
+protoc --version
+```
+
+Run these commands to find other protobuf installations that may be causing problems.
+
+```bash
+find /usr -name libprotobuf* 2>/dev/null
+find ~ -name libprotobuf* 2>/dev/null
+```
+Brew installs protobuf into `/usr/local` by default. Anaconda installs protobuf somehwere under the `anaconda` root folder (the command above assumes that you installed Anaconda into your home directory, as is recommended by the Anaconda).
```
Comment (orionr) [inline _docs/getting-started/mac.md]: Nit: somehwere -> somewhere

**Comment by pietern** [comment]
Hahaha, have to ignore the gh-pages branch on Jenkins ;)

File: _docs/getting-started/mac.md (hunk 1 above)
Comment (orionr) [reply _docs/getting-started/mac.md]: Nit: recommended by the Anaconda -> recommended by Anaconda

File: _docs/getting-started/mac.md [hunk 2]
```diff
@@ -113,7 +111,94 @@ Change to a different folder and test Caffe2 again. If you are using Anaconda or
 sudo install_name_tool -change libpython2.7.dylib ~/anaconda/lib/libpython2.7.dylib /usr/local/caffe2/python/caffe2_pybind11_state.so
 ```
 
-### Troubleshooting
+### Protobuf errors
+Protobuf version mismatch is a common problem. Having different protobuf
+versions often leads to incompatible headers and libraries.
+
+Run these commands to see which protbuf is your default (if you are using conda environments, then the current conda environment affects the output of these commands).
```
Comment (orionr) [inline _docs/getting-started/mac.md]: Nit: protbuf -> protobuf

**Review by orionr — APPROVED** [review]
Awesome! A few nits, but after fixing those we can merge.

**Pull request merged**